operator/pkg/images/images.go

```diff
@@ -0,0 +1,235 @@
+// Copyright DataStax, Inc.
+// Please see the included license file for details.
+
+package images
```

Comment:
I love splitting this out into its own package, and it's very unit testable 👍 👍 👍 👍 👍